### ISSUE TITLE
Deprecation warnings from sklearn

### ISSUE THREAD

**Issue by samuelstjean** [civisanalytics/python-glmnet#9]
The latest master threw me some warning such as

``` python
/home/samuel/anaconda3/lib/python3.5/site-packages/sklearn/cross_validation.py:44: 
DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved.
 Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.

  "This module will be removed in 0.20.", DeprecationWarning)
```

and

``` python
home/samuel/anaconda3/lib/python3.5/site-packages/sklearn/utils/validation.py:395: 
DeprecationWarning: Passing 1d arrays as data is deprecated in 0.17 and will raise ValueError in 0.19. 
Reshape your data either using X.reshape(-1, 1) if your data has a single feature or X.reshape(1, -1) if it contains a single sample.

  DeprecationWarning)
```

They are already at 0.18, so it's just annoying as of now but would be worth having a look into for the future.

**Comment by kcrum** [comment]
The top item in your issue is being addressed in #10, but the second part about 1d arrays does need a look.

**Comment by kcrum** [comment]
As #10 is now close, I'm going to close this issue. However please feel free to reopen is my comment about the 1d array deprecation warning did not actually address your issue.
